docs/scripts/defaults.js

```diff
@@ -1,15 +1,11 @@
-import {defaults, helpers} from 'chart.js';
+import {defaults} from 'chart.js';
+import {merge} from 'chart.js/helpers';
 
-const {merge} = helpers;
-
-merge(defaults.global, {
+merge(defaults.plugins, {
```

Comment:
We can use `defaults.set()` instead (as [here](https://github.com/chartjs/chartjs-plugin-datalabels/blob/master/docs/samples/defaults.js)):

```js
defaults.set({
  plugins: {
    legend: {
      display: false
    },
    tooltip: {
      enabled: false
    }
  }
});
```